src/Kanvas/Users/Repositories/UsersRepository.php

```diff
@@ -91,6 +91,14 @@ public static function getUserOfAppById(int $id, ?AppInterface $app = null): Use
             ->firstOrFail();
     }
 
+    public static function isUserInApp(int $id): bool
```

Comment:
@FredPeal we have a method already for this public static function belongsToThisApp